### ISSUE TITLE
Build config including sputnikvm

### ISSUE THREAD

**Issue by austinabell** [eth-classic/go-ethereum#15]
TL;DR: To build the project until all of the config is set up, run `make build WITH_SVM=0` which builds without the sputnikvm

So I have been looking into the config since the build has been working not as intended since the namespace move and move to go modules. 

Currently in the development branch, the build and running of geth will work by running:

```
make build WITH_SVM=0
```

With the current version, `make build` with assuming the sputnikvm config (since WITH_SVM defaults to 1) fails with clang errors such as one I ran into (but not limited to):

```
Undefined symbols for architecture x86_64:
  "_sputnikvm_out_copy_data", referenced from:
      __cgo_b01fc125d595_Cfunc_sputnikvm_out_copy_data in _x002.o
     (maybe you meant: __cgo_b01fc125d595_Cfunc_sputnikvm_out_copy_data)
  "_sputnikvm_out_len", referenced from:
      __cgo_b01fc125d595_Cfunc_sputnikvm_out_len in _x002.o
     (maybe you meant: __cgo_b01fc125d595_Cfunc_sputnikvm_out_len)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [cmd/geth] Error 2
```

Other errors I had run into had referenced multiple declarations of variables, also in the same process.

I was able to run with sputnikvm with go modules by editing all paths from the ETCDEVTeam/... to a cloned version where I edited the references in that repository to the eth-classic/go-ethereum since the ETCDEVTeam/sputnikvm-ffi references ethereumproject/go-ethereum (Also because their version is broken). I also changed the `build_sputnikvm.sh` file to access the go-ethereum code not from specifically the $GOPATH and also cloned the sputnikvm-ffi repository and referenced that folder to run that build.

A guess I have into why the c linker has been failing is with the -ldflags inside of the Makefile:
```
LDFLAGS=-ldflags "-X main.Version="`git describe --tags`
```
Since it is referenced in essentially every build command and the `git describe --tags` returns `fatal: No names found, cannot describe anything` since our cloned repository has no released versions that are tagged.  

As per a comment in the code:
```
// Version is the application revision identifier. It can be set with the linker
// as in: go build -ldflags "-X main.Version="`git describe --tags`
```

Depending on how you run the code with sputnikvm, it may add ethereumproject/go-ethereum as an indirect reference because ETCDEVTeam sputnikvm-ffi references the common package of go-ethereum. As I have mentioned before, go-ethereum had referenced the go code of sputnikvm-ffi and vice versa and the Rust build was run during the go-ethereum build to build the executable which is used in a flag during the build of geth specifically (at the end of the `build_sputnikvm.sh` file).
